cmd/resource.go

```diff
@@ -78,7 +78,7 @@ func listResourcesCmd(c *app.CLIConfig, adapter handlerv1beta1.ProtoAdapter) *co
 				return err
 			}
 
-			format := cmd.Flag("format").Value.String()
+			format := cmd.Flag("output").Value.String()
```

Comment:
@rahmatrhd does this denote the format of output? If yes, any reason to call it output and not format?

Reply:
I thought we are moving to this fashion `guardian appeals list -o json`. Few days ago we change it to `output` in some places >> https://github.com/odpf/guardian/pull/102/files#diff-16ff6f0fda308d588b6048ff5bf2a0a208113dadc5b190b36497f90178347f49R158. This is one where we forgot to replace

Reply:
IMHO in general `output` sounds less ambiguous than `format`.
Format could mean other things I guess 🤔 if no context provided.

Reply:
Yes, my bad. I changed it to output only for the same reason. :)